doc/introduction.rst

```diff
@@ -106,7 +113,8 @@ Core Contributors
   * **Nathan Keim** alternative trajectory-linking implementations, major
     speed-ups, prediction, adaptive search
   * **Thomas Caswell** multiple implementations of sophisticated trajectory-linking, tests
-  * **Casper van der Wel** anisotropic (3D) feature-finding, 3D plotting and analyses
+  * **Casper van der Wel** anisotropic 3D feature-finding, plotting and analyses, framework
```

Comment:
Feel free to put your name first in the list as the current most active maintainer.